libraries/AP_Math/tests/test_geodesic_grid.cpp

```diff
@@ -100,7 +100,7 @@ static bool section_triangle(unsigned int section_index,
                              Vector3f &b,
                              Vector3f &c) {
     if (section_index >= 80) {
-        return false;
+        return false;  // LCOV_EXCL_LINE
```

Comment:
These in the test suites are fine IMO.

Reply:
yep, another solution is to not track test_*.cpp files

Reply:
> yep, another solution is to not track test_*.cpp files

I think I'd rather have the magic comments in the test files.